templates/todo/projects/java-postgresql/.repo/terraform/infra/output.tf

```diff
@@ -0,0 +1,26 @@
+output "AZURE_POSTGRESQL_ENDPOINT" {
+  value     = module.postgresql.AZURE_POSTGRESQL_FQDN
+  sensitive = true
+}
+
+output "REACT_APP_WEB_BASE_URL" {
+  value = module.web.URI
+}
+
+output "REACT_APP_API_BASE_URL" {
+  value = module.api.URI
```

Comment:
should not the value reflect the case of ueAPIM = true, similar to here https://github.com/Azure/azure-dev/blob/127a291a300e723c086a44eee59bc049b4a5fc6f/templates/todo/projects/python-mongo/.repo/terraform/infra/output.tf#L19